req.go

```diff
@@ -783,10 +783,101 @@ type ReqFilter struct {
 	AttributeRegexp    map[string]*regexp.Regexp
 }
 
+// createFilterFromCmdLine reads the filter regular expressions from the command line arguments and
+// compiles them into a filter structure ready to use
+// @llr REQ-TRAQ-SWL-19, REQ-TRAQ-SWL-73
+func createFilterFromCmdLine(idFilter, titleFilter, bodyFilter string, attributeFilter []string) (ReqFilter, error) {
+	filter := ReqFilter{} // Filter for report generation
+	filter.AttributeRegexp = make(map[string]*regexp.Regexp, 0)
+	var err error
+	if len(idFilter) > 0 {
+		filter.IDRegexp, err = regexp.Compile(idFilter)
+		if err != nil {
+			return filter, err
+		}
+	}
+	if len(titleFilter) > 0 {
+		filter.TitleRegexp, err = regexp.Compile(titleFilter)
+		if err != nil {
+			return filter, err
+		}
+	}
+	if len(bodyFilter) > 0 {
+		filter.BodyRegexp, err = regexp.Compile(bodyFilter)
+		if err != nil {
+			return filter, err
+		}
+	}
+	if len(attributeFilter) > 0 {
+		for _, f := range attributeFilter {
+			if strings.Contains(f, "=") {
+				parts := strings.Split(f, "=")
+				r, err := regexp.Compile(parts[1])
+				if err != nil {
+					return filter, err
+				}
+				filter.AttributeRegexp[strings.ToUpper(parts[0])] = r
+			} else {
+				filter.AnyAttributeRegexp, err = regexp.Compile(f)
```

Comment:
we should probably trigger an error if the user specifies more than one `any` attribute.